svid/x509svid/tests/check_verify.c

```diff
@@ -63,11 +63,14 @@ START_TEST(test_x509svid_Verify)
     arrput(certs, leaf);
     arrput(certs, inter);
 
-    /// TODO: build bundle source
```

Comment:
I know this is not in the scope of this PR, but you could add some comments on each test case (or at least, in the test cases your are working in).